common/c_cpp/src/c/wombat/wtable.h

```diff
@@ -31,6 +31,8 @@ extern "C" {
 
 typedef void* wtable_t;
 
+/* Indicate the success of wTable functions. */
+#define WTABLE_INSERT_SUCCESS   1
```

Comment:
Yup - missed its use in mama.c because the preview didn't expand out the code on this page (!)